packages/cli/src/cli.ts

```diff
@@ -9,6 +9,7 @@ import { reportDiagnostics } from './report-diagnostics';
 const { join, resolve } = nodeFs;
 
 const argv = yargs
+    .usage('$0 [options]')
```

Comment:
what's the $0 means isn't it a command?

Reply:
it's the name of the executable that was run, so in most cases it will be `stc`